cli/src/pcluster/validators/efs_validators.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+from pcluster.validators.common import FailureLevel, Validator
+
+
+class EfsMountOptionsValidator(Validator):
+    def _validate(self, encryption_in_transit: bool, iam_authorization: bool):
+        if iam_authorization and not encryption_in_transit:
+            self._add_failure(
+                "IAM Authorization requires Encryption in Transit",
```

Comment:
Please make the error message more detailed.
The error message should describe the problem and the solution. Take a look at the ec2 validator here https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/validators/ec2_validators.py#L31.

Moreover, can you include the name of the share EFS here? Because it is possible to specify mulitiple EFS file systems, providing the name in the shared EFS will help users locate the misconfiguration in the cluster configuration file